module/models/action.mjs

```diff
@@ -237,6 +239,7 @@ export default class CrucibleAction extends foundry.abstract.DataModel {
         action: new fields.NumberField({required: true, nullable: false, integer: true, initial: 0}),
         focus: new fields.NumberField({required: true, nullable: false, integer: true, initial: 0}),
         heroism: new fields.NumberField({required: true, nullable: false, integer: true, initial: 0}),
+        hands: new fields.NumberField({required: false, nullable: false, integer: true, min: 0, initial: undefined}),
```

Comment:
I think we should do `initial: 0` here such that most actions don't require a free hand unless otherwise specified. Should also probably set `max: 2` although there might be special exceptions?
```suggestion
        hands: new fields.NumberField({required: false, nullable: false, integer: true, initial: 0, min: 0, max: 2}),
```